packages/pds/src/auth-verifier.ts

```diff
@@ -219,24 +229,39 @@ export class AuthVerifier {
     }
   }
 
-  adminService = async (reqCtx: ReqCtx): Promise<AdminServiceOutput> => {
-    const jwtStr = bearerTokenFromReq(reqCtx.req)
-    if (!jwtStr) {
-      throw new AuthRequiredError('missing jwt', 'MissingJwt')
-    }
-    const payload = await verifyServiceJwt(
-      jwtStr,
-      this.dids.entryway ?? this.dids.pds,
-      async (did, forceRefresh) => {
-        if (did !== this.dids.admin) {
-          throw new AuthRequiredError(
-            'Untrusted issuer for admin actions',
-            'UntrustedIss',
-          )
-        }
-        return this.idResolver.did.resolveAtprotoKey(did, forceRefresh)
+  userDidAuth = async (reqCtx: ReqCtx): Promise<UserDidOutput> => {
+    const payload = await this.verifyServiceJwt(reqCtx, {
+      aud: this.dids.entryway ?? this.dids.pds,
+      iss: null,
+    })
+    return {
+      credentials: {
+        type: 'user_did',
+        aud: payload.aud,
+        iss: payload.iss,
       },
-    )
+    }
+  }
+
+  userDidAuthOptional = async (
+    reqCtx: ReqCtx,
+  ): Promise<UserDidOutput | NullOutput> => {
+    if (isBearerToken(reqCtx.req)) {
+      try {
+        return await this.userDidAuth(reqCtx)
+      } catch {
+        return { credentials: null }
+      }
```

Comment:
I'd recommend we require the auth be valid if it's present.  It's conservative on the server side, but also usually friendly to the caller too, since if they included auth they probably intended to take action in the context of those credentials.  They would also be aware that they were holding onto some bad credentials.

Reply:
Yup that's fair 👍